src/AutoRest.CSharp/Common/Generation/Writers/JsonCodeWriterExtensions.cs

```diff
@@ -568,6 +572,11 @@ private static void DeserializeFrameworkTypeValue(CodeWriter writer, CodeWriterD
             writer.AppendRaw(")");
         }
 
+        private static bool IsMgmtReferenceType(Type frameworkType)
```

Comment:
nit, as this list may change, would like to keep this code in Mgmt rather than Common. Can we call out to a static checked to there?